src/lib/forms-validation/index.ts

```diff
@@ -138,6 +138,23 @@ function validateWithFormSchema(form?: unknown): FormTypes.Form {
     }
   }
 
+  if (
+    validatedForm.postSubmissionReceipt?.allowPDFDownload?.excludedElementIds
+  ) {
+    for (const elementId of validatedForm.postSubmissionReceipt.allowPDFDownload
+      .excludedElementIds) {
+      const element = formElementsService.findFormElement(
+        validatedForm.elements,
+        ({ id }) => id === elementId,
+      )
+      if (!element) {
+        throw new Error(
+          `You tried to reference an element (${elementId}) in "postSubmissionReceipt.allowPDFDownload.excludedElementIds" that does not exist on the form.`,
+        )
+      }
+    }
```

Comment:
I'm guessing we already do this, but is it such a problem if there are stray element ids in there?

Reply:
This validation is already applied to everywhere we reference an element.

This is a question for the product owner (or Monday Matt, he is very keen on code/product/process improvement, Thursday Matt is not so much).